src/main/java/nextstep/subway/line/domain/Section.java

```diff
@@ -53,4 +54,23 @@ public Station getDownStation() {
     public int getDistance() {
         return distance;
     }
+
+    public Section insertStation(Station downStation, int distance){
```

Comment:
```suggestion
    public void updateDownStation(Station downStation, int distance){
```
같은 객체를 한번 더 리턴해줘서 오히려 혼란스러운 것 같습니다.

return type은 void로 하고 해당 메소드 호출 후 기존 객체를 사용해보면 어떨까요? 😄 

추가적으로 별건 아니지만 [네이밍에 관련된 좋은 게시글](https://woowacourse.github.io/javable/post/2020-04-26-Method-Naming/)도 함께 공유드려 보겠습니다!